src/ResourceManager/SignalR/Commands.SignalR.Test/packages.config

```diff
@@ -14,8 +14,6 @@
   <package id="Microsoft.Bcl.Build" version="1.0.14" targetFramework="net452" />
   <package id="Microsoft.IdentityModel.Clients.ActiveDirectory" version="2.28.3" targetFramework="net452" />
   <package id="Microsoft.Net.Http" version="2.2.28" targetFramework="net452" />
-  <package id="Microsoft.Rest.ClientRuntime" version="2.3.11" targetFramework="net452" />
```

Comment:
see earlier comment - I think we should do this everywhere that uses common dependencies, or nowhere